scripts/manage_study.py

```diff
@@ -57,6 +57,14 @@
 # Avoid sending a user-agent string while obtaining the number of studies in SCP
 python manage_study.py --no-user-agent --token=$ACCESS_TOKEN list-studies --summary
 
+EXIT CODES
+79  incompatible scp-ingest-pipeline package version detected
+80  exit-file-already-exists-in-study-bucket
+81  exit-file-not-found-in-study-bucket
+82  exit-failed-to-gsutil-delete-file
+83  exit-uploaded-file-deleted
+84  exit-no-file-cleanup-needed
+85  exit-file-not-found-in-remote-bucket
```

Comment:
I don't consider it blocking, but now or later we should refactor to use Python errors instead of Bash errors.

As is, upon encountering a state we consider an error, this approach does:
* `print(specific_error_message)`
* `exit(specific_error_code`

That complicates downstream error handling in Python, e.g. logging the error to Sentry or Mixpanel.  Instead, raising a Python error (like we do in Ingest Pipeline [here](https://github.com/broadinstitute/scp-ingest-pipeline/blob/3f67f18ea761b8ffa920cce544e6ae61535dcbfc/ingest/expression_files/dense_ingestor.py#L167)) would ease writing to terminal, log file, and external services like Sentry or Mixpanel.  If custom exit codes are indeed needed, that can be done in Python error handling as shown [here](https://stackoverflow.com/a/6180231).

The current approach is fine for now (it doesn't cause any functional issues), but if you prefer to not refactor now then please open a tech debt ticket and note the refactoring as a TODO in the code.

Reply:
added SCP-2790 to backlog for future refactoring